src/algebra/module/submodule_lattice.lean

```diff
@@ -247,3 +247,51 @@ add_submonoid.to_nat_submodule.apply_symm_apply S
 end nat_submodule
 
 end add_comm_monoid
+
+
+section int_module
+
+variables [add_comm_group M]
+
+instance M_int_module : module ℤ M := add_comm_group.int_module M
```

Comment:
```suggestion
```
This doesn't do anything because it's already flagged as an instance

Reply:
Thank you